### PR TITLE
Bump website from 3.0 to 4.

### LINKED ISSUE #4020: inspec website still referring to 3.0
Changing to "4" so we don't have to bump on each minor.

I need the bullet points for 4's release to be able to finish. Who do I bug about that?

### PR DESCRIPTION
**Pull request by zenspider**
Specifically using "4" so we don't have to bump on every minor
release.

I mish-mashed the bullet points from 2 different sources. It is
probably wrong or confusing and I'd like extra scrutiny there.

I don't remember slim AT ALL and probably got some stuff wrong.

This should fix #4020.

Signed-off-by: Ryan Davis <zenspider@chef.io>

### CODE REVIEW

File: www/source/index.html.slim [hunk 1]
```diff
@@ -27,24 +27,22 @@ header.bg-gradient.margin-top-offset.short-bg.relative
   /! announcement
   .row.columns
     .box-white.shadow.z-20.slide-left
-      h3.large-11.medium-11 Announcing InSpec 3.0
-      p.large-11.medium-11 Plugin system, global attributes, enchanced skip messaging, and more.
+      h3.large-11.medium-11 Announcing InSpec 4
+      p.large-11.medium-11 Numerous bugfixes and new features.
```
Comment (clintoncwolfe) [inline www/source/index.html.slim]: "Inputs system overhaul, updated AWS-SDK to v3, support for the new Chef License, and many bugfixes and minor features"

File: www/source/index.html.slim [hunk 2]
```diff
@@ -27,24 +27,22 @@ header.bg-gradient.margin-top-offset.short-bg.relative
...
         .margin-top-sm.brdr-left.large-11.medium-11
           .margin-left-sm
             h4.t-purple core
-            p New plugin system for InSpec CLI and Train
-            p Enhanced "only_if" skip messaging
-            p Unsupported profiles are now hidden from output
-            p Attributes are now specified in <a href="/docs/reference/profiles/">inspec.yml</a>
-            p Text based severity (e.g. "impact 'low'" )
-            p Tons of security/bug fixes and improvements
+            p Added license acceptance (see below)
```
Comment (clintoncwolfe) [inline www/source/index.html.slim]: p Inputs system overhaul - more consistent, extensible, and predictable
p Updated AWS-SDK to v3
p Laying the foundation for opt-in telemetry
p Support for the new Chef License
p Add support for `more_permissive_than?` on the File resource
p many other minor bugfixes!

File: www/source/index.html.slim [hunk 3]
```diff
@@ -27,24 +27,22 @@ header.bg-gradient.margin-top-offset.short-bg.relative
...
+            p Updated AWS profile generation
         .margin-top-sm.brdr-left.large-11.medium-11
           .margin-left-sm
-            h4.t-purple cloud
-            p
-              | New <a href="http://www.inspec.io/docs/reference/resources/#azure-resources">Azure</a> and <a href="/docs/reference/resources/#gcp-resources">GCP resources</a>
-            p Support for VMware ESXi via PowerCLI (Even on Linux!)
+            h4.t-purple inspec-bin
+            p Moved the `inspec` executable to a separate gem, `inspec-bin`.  This allows use of the `inspec` gem as a library without the licensing restrictions placed on the executable.
+            p This release is the first to require accepting a license before using it.
```
Comment (clintoncwolfe) [inline www/source/index.html.slim]: ...using it as an executable.

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Just some minor wording changes, related to changelog

File: www/source/index.html.slim (hunk 3 above)
Comment (zenspider) [reply www/source/index.html.slim]: done done and done

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

**Review by miah — APPROVED** [review]

File: www/source/index.html.slim [hunk 4]
```diff
@@ -27,24 +27,23 @@ header.bg-gradient.margin-top-offset.short-bg.relative
...
-            p Unsupported profiles are now hidden from output
-            p Attributes are now specified in <a href="/docs/reference/profiles/">inspec.yml</a>
-            p Text based severity (e.g. "impact 'low'" )
-            p Tons of security/bug fixes and improvements
+            p Inputs system overhaul - more consistent, extensible, and predictable
+            p Updated AWS-SDK to v3
+            p Laying the foundation for opt-in telemetry
+            p Support for the new Chef License
+            p Add support for more_permissive_than? on the File resource
+            p many other minor bugfixes!
```
Comment (mjingle) [inline www/source/index.html.slim]:
```suggestion
            p Many other minor bugfixes!
```

File: www/source/index.html.slim [hunk 5]
```diff
@@ -27,24 +27,23 @@ header.bg-gradient.margin-top-offset.short-bg.relative
...
         .margin-top-sm.brdr-left.large-11.medium-11
           .margin-left-sm
-            h4.t-purple cloud
-            p
-              | New <a href="http://www.inspec.io/docs/reference/resources/#azure-resources">Azure</a> and <a href="/docs/reference/resources/#gcp-resources">GCP resources</a>
-            p Support for VMware ESXi via PowerCLI (Even on Linux!)
+            h4.t-purple inspec-bin
+            p Moved the `inspec` executable to a separate gem, `inspec-bin`.  This allows use of the `inspec` gem as a library without the licensing restrictions placed on the executable.
```
Comment (mjingle) [inline www/source/index.html.slim]:
```suggestion
            p Moved the `inspec` executable to a separate gem, `inspec-bin`. This allows the use of the `inspec` gem as a library without the licensing restrictions placed on the executable.
```

**Comment by mjingle** [comment]
Just a few copyedits and then it looks good to go!

**Comment by zenspider** [comment]
Thank you @clintoncwolfe and @mjingle! Applied and awaiting CI.

**Comment by mjingle** [comment]
Ack, committing suggestions via comments does not allow for a DCO to be applied in the process. 👎

**Comment by clintoncwolfe** [comment]
As this is a website change, I'm going to ignore the CI failures.

**Pull request merged**